tiles/src/main/java/com/protomaps/basemap/layers/Housenumbers.java

```diff
@@ -0,0 +1,34 @@
+package com.protomaps.basemap.layers;
+
+import com.onthegomap.planetiler.FeatureCollector;
+import com.onthegomap.planetiler.ForwardingProfile;
+import com.onthegomap.planetiler.VectorTile;
+import com.onthegomap.planetiler.geo.GeometryException;
+import com.onthegomap.planetiler.reader.SourceFeature;
+import com.protomaps.basemap.feature.FeatureId;
+
+import java.util.List;
+
+public class Housenumbers  implements ForwardingProfile.LayerPostProcesser {
+  public void processOsm(SourceFeature sf, FeatureCollector features) {
+    if (sf.hasTag("addr:housenumber")) {
+      if (sf.isPoint()) {
```

Comment:
Also a lot of ways in osm have a house number: https://taginfo.openstreetmap.org/keys/addr%3Ahousenumber#overview

Reply:
Yep, I agree. As I mentioned in PR description I wanted to handle them as well, but wanted to first agree on the general approach. Do you think such implementation does have a chance to be eventually merged? 😀 Ofc after handling all the cases, adding tests etc...

Reply:
I think it is a good idea to add house numbers.

Reply:
This looks like a great start! We should align the layer name and properties to conform with the Tilezen docs: https://tilezen.readthedocs.io/en/latest/layers/#buildings-and-addresses

so it should be points included in the `buildings` layer with a property called `addr_housenumber`